cdap-data-fabric/src/test/java/co/cask/cdap/api/dataset/lib/SystemDatasetDefinitionTest.java

```diff
@@ -100,13 +101,13 @@ private void testTimeSeriesReconfigure(DatasetDefinition def) throws Incompatibl
       .add(TimeseriesDataset.ATTR_TIME_INTERVAL_TO_STORE_PER_ROW,
            String.valueOf(TimeUnit.HOURS.toMillis(1)))
       .build();
-    DatasetProperties compatProps = DatasetProperties.builder()
+    DatasetProperties compatProps = TableProperties.builder()
+      .setTTL(TimeUnit.DAYS.toSeconds(1))
```

Comment:
assuming the change in unit was intentional because the test was never updated when ttl was moved to seconds?